### DISCUSSION TITLE
Tic Tac Toe game

### DISCUSSION THREAD

**Discussion by VeryUsual** [InfiniTimeOrg/InfiniTime#1735] [Ideas]
This would be very cool.

If you don't know what tic tac toe is it is this:
Tic-tac-toe is a paper-and-pencil game for two players who take turns marking the spaces in a three-by-three grid with X or O. The player who succeeds in placing three of their marks in a horizontal, vertical, or diagonal row is the winner. It is a solved game, with a forced draw assuming best play from both players.

**Comment by 88572** [comment]
+1
